soc/arm/Kconfig

```diff
@@ -20,3 +20,12 @@ config CPU_HAS_ARM_SAU
 	select CPU_HAS_TEE
 	help
 	  MCU implements the ARM Security Attribution Unit (SAU).
+
+config CPU_HAS_NRF_IDAU
+	bool
+	# Omit prompt to signify "hidden" option
+	depends on SOC_SERIES_NRF91X
+	select CPU_HAS_TEE
+	help
+	  MCU implements the nRF (Implementation-defined)
```

Comment:
`(Implementation defined)` after nRF does not look well.
What about "MCU implements Nordic flavour of Security... (IDAU)."

Reply:
new text:

> MCU implements the nRF (vendor-specific) Security Attribution Unit.
 (IDAU: "Implementation-Defined Attribution Unit", in accordance with
  ARM terminology).

Hope this clarifies it better, @pizi-nordic